java/core/src/main/java/sleeper/core/statestore/PartitionStore.java

```diff
@@ -69,5 +69,8 @@ void atomicallyUpdatePartitionAndCreateNewOnes(Partition splitPartition,
      */
     void initialise(List<Partition> partitions) throws StateStoreException;
 
-    void clearTable();
+    /**
+     * Clears all partition data from the partition store.
```

Comment:
Add "Note that after calling this method the PartitionStore must be initialised before the Sleeper table can be used again." ?

Reply:
Since FileInfoStore.clearSleeperTable and PartitionStore.clearSleeperTable have different Javadoc, maybe they should have different names as well, and we can add a method on StateStore that calls both? Right now that's handled in DelegatingStateStore.

Reply:
That sounds good